### ISSUE TITLE
Installer stops at EXITBS:START

### ISSUE THREAD

**Issue by HackyMacky** [luchina-gabriel/EFI-X99-HUANANZHI-F8-INTEL-XEON-V4-RX-580-8GB#1]
Hi Gabriel, thanks for this great repo. I have the same Huananzhi X99 F8 board with below specs:

- Sapphire RX570 4B
- 64GB ECC RAM (8x8GB)
- E5-2680 V4 
- Attempting Monterey installation

I tried your EFI and it it always boots and stops at EXITBS:START. I have also build my own ssdts and EFI using oc 0.7.9 but also get the same issue. 

Do I have to flash a custom bios for the board before I can get this working?

Here is a [screenshot](https://i.imgur.com/4sAZBcD.jpg) of the boot. Here's the [debug log](https://drive.google.com/file/d/1kSkPTG_CyddA8itv_RIfgaRh3LyoF2qT/view?usp=drivesdk) in case you need it. 

Hoping to get this working.

**Comment by HackyMacky** [comment]
Update: I flashed the MacOS bios (non turbo non undervolt) from your youtube files repo and the issue still persists. So I'm guessing it now narrows down to either the cpu or the gpu. Will try to troubleshoot further.

**Comment by HackyMacky** [comment]
Update: So apparently although its the same Huananzhi X99 F8 motherboard, mine is an older revision with Nuvoton NCT5532D. The new revision for 2021 comes with Nuvoton NCT5567D-B. I flashed my bios with BIOS-iEngineer CX99DE28 bios and I was able to install Monterey 12.3 successfully. Maybe this will help anyone else with the same older Huananzhi X99 F8 motherboard. Thanks Gabriel for your great work.
